helm/diffcalc-api/values.yaml

```diff
@@ -103,6 +103,18 @@ mongodb:
   persistence:
     existingClaim: diffcalc-mongo-volume
     createNew: false
+  livenessProbe:
```

Comment:
Looking at the template values, I think you need `customLivenessProbe`

https://github.com/bitnami/charts/blob/main/bitnami/mongodb/values.yaml#L536

Reply:
All done now :+1: